src/pylife/materialdata/woehler/elementary.py

```diff
@@ -129,7 +129,7 @@ def _transition_cycles(self, fatigue_limit):
         return 10**(self._lg_intercept + self._slope * (np.log10(fatigue_limit)))
 
     def _pearl_chain_method(self):
-        self._pearl_chain_estimator = PearlChainProbability(self._fd.fractures, self._slope)
+        self._pearl_chain_estimator = PearlChainProbability(self._fd.finite_zone.loc[self._fd.finite_zone.fracture == True], self._slope)
```

Comment:
Please consider to store `self._fd.finite_zone.loc[self._fd.finite_zone.fracture == True]` in a class variable – e.g. `self._finite_fractures` – in `.__init__()`, so that you don't have to write it out three times.

Reply:
Thanks for the tip. I will definitely add this new class variable to the code.

Reply:
I had to add the class variable to the `analyze` function. Here there is a check for "Need at least two load levels to do a Wöhler analysis." In `.__init__()` the class variable would case issues.